buildSrc/src/main/kotlin/packetevents.shadow-conventions.gradle.kts

```diff
@@ -20,9 +20,45 @@ tasks {
         mergeServiceFiles()
     }
 
+    create<ShadowJar>("shadowNoAdventure") {
+        group = rootProject.name
+        description = "Create a combined JAR of project and runtime dependencies without Adventure dependencies."
+        archiveFileName = "packetevents-${project.name}-no-adv-${rootProject.ext["versionNoHash"]}.jar"
+        archiveClassifier = null
+
+        val shadowJar = shadowJar.get()
+        val sourceSets = project.extensions.getByType<SourceSetContainer>()
+
+        manifest.inheritFrom(shadowJar.manifest)
+
+        from(sourceSets.main.get().output)
+        configurations = shadowJar.configurations
+
+        relocate("net.kyori.adventure.text.serializer", "io.github.retrooper.packetevents.adventure.serializer")
+        relocate("net.kyori.option", "io.github.retrooper.packetevents.adventure.option")
+
```

Comment:
`relocate("org.bstats", "io.github.retrooper.packetevents.bstats")`

The bstats relocation is missing here

Reply:
Has this been resolved?

Reply:
It should be added after the merging of this pr to preserve a better commit history